docusaurus/docs/iOS/swiftui/audio-recording.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Voice Recording
+---
+
+Stream Chat's SwiftUI SDK allows you to record and share async voice messages in your channels. The voice recordings have a built-in attachment type (as defined [here](https://getstream.io/chat/docs/sdk/ios/uikit/guides/working-with-attachments/)).
+
+:::note
+Voice Recordings on SwiftUI are available since version 4.46.0.
```

Comment:
What do you think about adding a link to the release on GitHub (I know it's not released yet, but the link will work after releasing)

Reply:
done